### PR TITLE
feat(TCWI-1018) - UI coding test, Tile view

### PR DESCRIPTION
**Pull request by lus21**
[Story Link](https://tbol-com.atlassian.net/browse/TCWI-1018)

### Screenshots

**Tile view**

image

**Numbers** - /numbers

image

**Kanye** - /kanye

image

**Morty** - /morty

image

**Items** - /items

image

**404** 

image

**for 1400px**
image

### CODE REVIEW

File: src/zz_TestRunner/Descriptions/CounterDescription/CounterDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const CounterDescription: React.FunctionComponent = () => {
+  return (
+    <div>
+      <p>
+        Open the Counter Component -<b>src/components/tests/Counter/Counter.tsx</b>
+      </p>
+      <p>It's is displayed on the right side.</p>
```
[the remark this answers is not in the document]
Comment (lus21) [reply src/zz_TestRunner/Descriptions/CounterDescription/CounterDescription.tsx:9]: Right now the page looks like this. 
image
There is no question or task or an instruction. 
 @NiklasMencke @varuntbol @jackson-miller-tbol 
 Should any instruction be added and if so what kind of instruction should it be?

**Comment by lus21** [comment]
> Also, Lucy, if when you were making this you had ideas about what we should be looking for in the candidate's work, can you post that here. We'll be making a more in-depth interviewer's guide on this and want to include things to look for

- Numbers - Expected that candidate change the Class component to Functional, to remove unused types, to prevent the counter from having a negative value. 
This task will help to see if the candidate is able to work with Class and Functional Components, is familiar with the latest React features, is caring for code quality.
- Kanye - Expected that candidate fetch simple data from the API, handle both success and error cases, and display it with some styling.
 This task will help to see if the candidate is able to work with a simple API, can do a request, is caring for different outcomes, is caring for styling, is able to use hooks.
- Morty -  Expected that candidate fetch not simple data from the API, handle both success and error cases, and display it with some styling.  The data is an object with nested objects and an array of objects.  Expected that candidate handle all this data and show it in a user-friendly way, for example as a table. Data contains image URLs, so the interviewer can see how the images are handled, for example, is alt field filled, how sizes are set, and is loading state handled. Also, the task will allow seeing if the candidate caring for different parameters such as the parameter for limiting results and is a candidate going for pagination.
- Items - Expected that the candidate create a form with validation and submit functionalities. The candidate should be able to work with different types of inputs such as text, number, select. 
This task will show is a candidate able to work with forms, cares for validation, handles the various responses, and handles the loading state while data is being submitted.

File: src/components/Descriptions/FormDescription/FormDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            <li>Clothing</li>
+            <li>Jewelery</li>
+            <li>Electronics</li>
+          </ul>
+        </li>
+      </ul>
+      <p>
+        Description and image are optional fields, all other fields are
+        required. Have validation for the form. Submit it to{" "}
+        <i>https://fakestoreapi.com/products</i> with POST method. See the API{" "}
```
[the remark this answers is not in the document]
Comment (jackson-miller-tbol) [reply src/components/Descriptions/FormDescription/FormDescription.tsx]: I think that works very well, we wouldnt want someone to use either of those methods

**Comment by jackson-miller-tbol** [comment]
While we don't need this to be fully responsive, there are some issues at 1400px, can we add some wrapping to the instructions?  This occurred in an interview I had today
image

**Comment by lus21** [comment]
> While we don't need this to be fully responsive, there are some issues at 1400px, can we add some wrapping to the instructions? This occurred in an interview I had today image image

image

**Review by jackson-miller-tbol — APPROVED** [review]

**Comment by jackson-miller-tbol** [comment]
> There is an issue on linux where a dependency causes a failure. Will post more about this later

FYI I pushed a change that documents the fix for the bug and updated react-scripts which maybe fixes it

**Pull request merged**